noxfile.py

```diff
@@ -74,3 +74,18 @@ def dev(session: nox.Session) -> None:
 
     python = os.path.join(venv_dir, "bin/python")
     session.run(python, "-m", "pip", "install", "-e", ".[dev]", external=True)
+
+
+@nox.session(python=["3.9", "3.10", "3.11", "3.12", "pypy3.9", "pypy3.10"])
+def examples(session: nox.Session) -> None:
+    session.install(".[examples]")
+    try:
+        session.install("unstructured[pdf]")
```

Comment:
can it be simplified now (move into the regular requirements?)

Reply:
No, it does not build for all combinations of OS + Python versions. [example](https://github.com/iterative/datachain/actions/runs/10276130816/job/28436039948?pr=199). This is why we need `can_import_unstructured`.

Reply:
Hmm, I don't see any error (specific to this) in that link 🤔 - can you share a specific line in the logs / job?

Reply:
https://github.com/iterative/datachain/actions/runs/10276130816/job/28436039948?pr=199#step:6:19 - unstructured depends on pycrypto which doesn't build.

Reply:
ah, I see this is `uv` bug! add `pdfplumber==0.11.1` as a dependency to force uv to use a proper version of it